Sources/Liquid/Variable.swift

```diff
@@ -36,8 +36,24 @@ struct Variable {
 	
 	private func parseFilterArgs(_ parser: Parser) -> [Expression] {
 		var args: [Expression] = []
-		
+
 		while !parser.look(.endOfString) {
+			// Assuming all args are named parameters
+			if parser.look(.id) && parser.look(.colon, 1) {
+				guard let key = parser.consume(.id) else { fatalError("id should exist") }
```

Comment:
```suggestion
				let key = parser.consume(.id)!
```
if we are going to fatalError than a force unwrap is the same thing (plus we are already guaranteed from the if check that this is true, so we'll just satisfy compiler)